persistence-jdbc/core/src/main/scala/com/lightbend/lagom/internal/persistence/jdbc/SlickProvider.scala

```diff
@@ -179,9 +179,18 @@ private[lagom] class SlickProvider(
         }
       } else DBIO.successful(Some(schema))
     }
-
   }
 
+  // Some older JDBC drivers don't implement Connection.getSchema
+  // (including some builds of H2). This causes them to throw an
+  // AbstractMethodError at runtime.
+  // Because Try$.apply only catches NonFatal errors, and AbstractMethodError
+  // is considered fatal, we need to construct the Try explicitly.
+  private def tryGetSchema(connection: Connection): Try[String] =
```

Comment:
With this workaround, it isn't required. As it turns out, the version of H2 we ship with Lagom 1.2.x does not implement the method (hence this change) and the version of H2 we ship with Lagom 1.3.x does implement it but always returns `null`. The latest version of H2 does implement it, but the latest version of the MySQL driver also always returns `null`. 😩